src/app/screens/Bitcoin/ConfirmSignPsbt/index.tsx

```diff
@@ -112,7 +112,7 @@ function ConfirmSignPsbt() {
             </div>
             <div className="flex w-full justify-center">
               <Hyperlink onClick={toggleShowAddresses}>
-                {showAddresses ? t("hide_details") : t("view_details")}
+                {showAddresses ? tCommon("details") : tCommon("details")}
```

Comment:
Remove this conditional as it yields the same output in any case.

Reply:
resolved

Reply:
yes as you can see onClick  toggleShowAddress is triggered which decides popup is shown or not